frontend/src/components/popup/Filter.tsx

```diff
@@ -65,9 +66,21 @@ const filterSpecs: { [k in Search.FilterKey]: FilterSpec } = {
 export default memo(function Filter() {
     const setPopup = useStore((store) => store.setPopup);
     const addFilter = useStore((store) => store.addSearchFilter);
+    const showOnlyNonConflicting = useStore((store) => store.showOnlyNonConflicting);
+    const setShowOnlyNonConflicting = useStore((store) => store.setShowOnlyNonConflicting);
 
     return (
         <div>
+            <div>
+                <Slider
+                    value={showOnlyNonConflicting}
+                    text="show only non-conflicting sections"
```

Comment:
Instead of "show only non-conflicting sections", it's probably better to say "hide conflicting sections" or something like that. Also, the text on the legacy frontend says "Hide courses conflicting with schedule", and we can probably just keep that.